core/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+use merlin::Transcript;
+use std::convert::TryFrom;
+
+pub struct BitcoinOutputLink {
```

Comment:
in bitcoin it's called "Outpoint", we can use the same term here.

Cf. https://github.com/bitcoin/bitcoin/blob/816e15ee81a2029cde6b4f9fe6fb93e75478c903/src/primitives/transaction.h#L26

Reply:
Fixed in 340a047.